taca/utils/bioinfo_tab.py

```diff
@@ -173,7 +176,6 @@ def get_ss_projects(run_dir):
             #This is a non platform MiSeq run. Disregard it.
             return []
         data=ss_reader.data
-
     else:
         csvf=open(FCID_samplesheet_origin, 'rU')
```

Comment:
@sylvinite why not use samplesheet parser implemented in flowcell_parser? https://github.com/SciLifeLab/flowcell_parser/blob/master/flowcell_parser/classes.py#L213

equivalent but in this way you can reuse code